Dockerfile.prod

```diff
@@ -0,0 +1,9 @@
+FROM nginx:1.14
+
+EXPOSE 80 9000
```

Comment:
is anything listening to port 9000 in this container? Why are we exposing it?

Reply:
Removed 9000 port. Don't know why it was there